packages/main/src/themes/Option.css

```diff
@@ -0,0 +1,7 @@
+:host {
+	height: var(--_ui5_list_item_dropdown_base_height);
+}
+
+:host ::part(icon) {
```

Comment:
this one is strange. you don't have icon part defined in your option.hbs file. why do we need it?